specification/2.0/README.md

```diff
@@ -1086,7 +1086,7 @@ This is illustrated in the following figure, where the respective UV coordinates
 <img src="figures/texcoords.jpg" /><br/>
 </p>
 
-Any colorspace information (such as ICC profiles, intents, etc) from PNG or JPEG containers must be ignored.
+Any colorspace information (such as ICC profiles, intents, etc) from PNG or JPEG containers must be ignored. Effective transfer function is defined by a glTF object that refers to the image.
```

Comment:
I may be overthinking this, but... any glTF object?  Only a material, or material extension?  Will viewers ever get into decoding troubles if the same image is referenced with two different encodings (for example a BaseColor sharing an image with MetallicRoughness)?  Do viewers ever have to "search" the material list for this reference, and would it help them to halt that search as soon as the first reference to the image is found?  If so, maybe a clarification that any given image can be referenced multiple times but only as a single colorspace, so sRGB material components can't share an image/texture with linear material components?

Reply:
So far, _images_ are referred by texture objects (from the core spec) and `EXT_lights_image_based.light.specularImages[]`.

> BaseColor sharing an image with MetallicRoughness

This sounds like a possible validation warning. There could be two material slots with different transfer functions pointing to the same texture object; or two texture objects (used by two material slots with different transfer functions) pointing to the same image.

Reply:
Right, I'm asking if the spec itself should state that the model can't implicate multiple transfer functions (both linear and sRGB) onto the same image, regardless of the indirection through shared or non-shared texture objects.  Would that be beneficial to apps trying to determine the transfer function?  I don't know, but reading the linked issue makes it sound like this is desirable.

Reply:
I'm not sure about explicitly disallowing such edge case. Currently, it's possible to pack all textures into one and use multiple UVs or texture transforms to get proper texels. Multiple transfer functions can be handled by keeping two copies of the image on the GPU, or by applying correction in the fragment shader (less accurate). I doubt that engines will handle this case properly, though.

Reply:
Interesting.  I suppose it's easy enough to create a test model with this arrangement.  Doesn't sound like we should be encouraging it though.